src/store/modules/auth.js

```diff
@@ -0,0 +1,95 @@
+import {
+    clearAccessToken,
+    getAccessToken,
+    getLoginUrl,
+    getParameterByName,
+    getSignupUrl,
+    setAccessToken
+} from '../../services/auth';
+import client from '../../services/client';
+import router from '../../router';
+
+const state = {
+    token: getAccessToken(),
+    user: null,
+};
+
+const mutations = {
+    LOGIN(state, {token}) {
+        state.token = token;
+    },

```

Comment:
Your indentation is so inconsistent. Here it's 4 spaces, other files it's 2 spaces, others 1 tab character. Please make this all consistent at 2 spaces for JS/html/css